trappy/ftrace.py

```diff
@@ -76,6 +108,22 @@ def __init__(self, name="", normalize_time=True, scope="all",
             setattr(self, attr, trace_class)
             self.trace_classes.append(trace_class)
 
+        cache_path = _trace_cache_path(self.trace_path)
+        cache_good = False
+
+        if _check_trace_cache(self.trace_path):
+            cache_good = True
+            try:
+                # Read csv into frames
+                for trace_class in self.trace_classes:
+                    csv_file = cache_path + '/' + trace_class.unique_word[:-1] + '.csv'
+                    trace_class.read_csv(csv_file)
+            except:
```

Comment:
Maybe we should log a warning here? Would be annoying if we later broke this code and didn't notice for ages because an `AttributeError` was getting thrown away or something.